### PR TITLE
Updated/Modified angle.html

### PR DESCRIPTION
**Pull request by Graru1**
1.2.1 Added Biggest Angle Difference

### CODE REVIEW

File: angle.html [hunk 1]
```diff
@@ -25,10 +25,23 @@
...
 			// Check special starts by default, 0 = Off, 1 = On
 			var u_check_special_start = 1;
+			
+			// Check biggest angle difference by default, 0 = Off, 1 = On
+			var u_check_biggest_diff = 1;
 		
 		/* END USER CONFIG */
 	</script>
 	
+    <!-- Bootstrap core CSS -->
```
Comment (mkrl) [inline angle.html]: Bootstrap core CSS and jquery is not required as this file is being bundled in the layout automatically. No styles or scripts are necessary.

File: angle.html [hunk 2]
```diff
@@ -25,10 +25,23 @@
...
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+	<link href="css/bootstrap-select.min.css" rel="stylesheet">
+
+    <!-- Custom styles for this template -->
+    //<link href="css/jumbotron-narrow.css" rel="stylesheet">
+	
+	<script type="text/javascript" src="js/jquery-2.1.4.min.js" ></script>
+	<script type="text/javascript" src="js/bootstrap.min.js" ></script>
+
```
Comment (mkrl) [inline angle.html]: This entire section should be removed.

**Review by mkrl — CHANGES_REQUESTED** [review]

**Comment by mkrl** [comment]
LGTM, thanks!

File: angle.html (hunk 1 above)
Comment (Graru1) [reply angle.html]: Removed Bootstrap core CSS and jquery

File: angle.html (hunk 2 above)
Comment (Graru1) [reply angle.html]: Removed Bootstrap core CSS and jquery

**Pull request merged**